test/realtimecontroller/DefaultRealtimeController.test.ts

```diff
@@ -548,6 +548,16 @@ describe('DefaultRealtimeController', () => {
       expect(state.volumeIndicatorCallbacks).to.equal(expectedResult);
     });
 
+    it('will not error if unsubscribed from a non-exist attendeeId', () => {
+      const callback = (
```

Comment:
nit: You should be able to have a simple callback, i.e., `const callback = () => {}`.